### PR TITLE
Add article on Python Scientific Stack

### PR DESCRIPTION
**Pull request by rpavelka**
Preview available here:
https://github.com/rpavelka/content/blob/52cbc72d562cddd872082b1df81de7286f1c01c3/tech/languages/python/scipy.md

### CODE REVIEW

**Comment by frenzymadness** [comment]
Hello.

If you have nothing against, could you please merge it soon. We are preparing site where we want to link to this article on the developer portal.

Thank you very much and have a nice day.
Lumír

**Comment by phracek** [comment]
I will care about it as I have promised.

File: tech/languages/python/scipy.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+* [SymPy](http://www.sympy.org/en/index.html), a Python library for computer algebra
+  support (i.e. symbolical computation) and
+* [nose](http://nose.readthedocs.io/en/latest/), a testing framework for Python.
+
+## Numpy
+
+[NumPy](http://www.numpy.org/) is a high performance Python library providing
+fast multidimensional arrays featuring vector operations. What are vector
+operations? Those which are applied on multiple values at once, see:
+
```
Comment (phracek) [inline tech/languages/python/scipy.md]: I would prefer to move there an installation via `dnf` and afterwards explain how to work with `Numpy`

File: tech/languages/python/scipy.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+```python
+>>> import scipy.integrate
+>>> scipy.integrate.quad(lambda x: 2*x, 0, 1)
+(1.0, 1.1102230246251565e-14)
+```
+
+## matplotlib
+
+[matplotlib](http://matplotlib.org/) is a graph plotting library producing publication-quality plots. See
```
Comment (phracek) [inline tech/languages/python/scipy.md]: `See ...` sentence should go to the new line.

File: tech/languages/python/scipy.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+[showcases gallery](http://matplotlib.org/gallery.html), e.g.:
+
+![histogram subplot example](http://matplotlib.org/mpl_examples/pylab_examples/scatter_hist.png)
+
+![3dplot example](http://matplotlib.org/mpl_examples/mplot3d/contour3d_demo3.png)
+
+Installation is simple:
```
Comment (phracek) [inline tech/languages/python/scipy.md]: This should be moved at the beginning or after description.

File: tech/languages/python/scipy.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+
+```bash
+$ sudo dnf install python3-matplotlib
+```
+
+Test it by
```
Comment (phracek) [inline tech/languages/python/scipy.md]: This should go before the pictures. As it is an example.

**Comment by phracek** [comment]
@jhornice Can you please look at the english?

**Comment by rpavelka** [comment]
OK, I have quite a lot of more urgent work up to ca 16:00 and then I will address it...

**Comment by rpavelka** [comment]
Addressed from pyvo, thanks to @mbag for corrections...

**Comment by hroncok** [comment]
Hi, could we please see it on https://developer.fedoraproject.org/ ?

**Pull request merged**